src/controller/task.spec.ts

```diff
@@ -0,0 +1,215 @@
+import { expect } from 'chai'
+import { after, before, describe } from 'mocha'
+import TaskModel from '../model/task'
+import axios from 'axios'
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import app from '../app'
+
+const _server = app
+axios.defaults.baseURL = 'http://localhost:3000'
+
+const Tasks = {
```

Comment:
This is just object so it should be in camelCase format. like testTask etc.

Reply:
fixed 6348b91338126d857fa44ff466646d2a6e09c839